sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -1785,6 +1786,53 @@ async def test_queue_async_send_dict_messages(self, servicebus_namespace_connect
                         received_messages.append(message)
                 assert len(received_messages) == 6
 
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    @ServiceBusQueuePreparer(name_prefix='servicebustest')
+    async def test_queue_async_send_mapping_messages(self, servicebus_namespace_connection_string, servicebus_queue, **kwargs):
```

Comment:
out of curiosity, is there only a test here because the changes made for Mapping were the same everywhere? so if this tests passes/fails, it applies to the mgmt `update_*` as well?

Reply:
Yeah - my thinking was that the approach I took was the same for both messages as properties:
```
try:
    obj_type(**random_value)
except TypeError:
    ...
```
We already had coverage for objects and dictionaries across all scenarios. I mostly wanted to affirm that this pattern could handle a non-dict Mapping.

Reply:
ah, that makes sense, thank you!